src/main/java/uk/gov/hmcts/divorce/solicitor/service/task/SetConfirmServiceDueDate.java

```diff
@@ -2,45 +2,25 @@
 
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
 import uk.gov.hmcts.divorce.common.service.HoldingPeriodService;
 import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
-import uk.gov.hmcts.divorce.divorcecase.model.SolicitorService;
 import uk.gov.hmcts.divorce.divorcecase.model.State;
 import uk.gov.hmcts.divorce.divorcecase.task.CaseTask;
 
-import static org.springframework.util.CollectionUtils.isEmpty;
-
 @Component
 @Slf4j
 public class SetConfirmServiceDueDate implements CaseTask {
 
-    @Value("${aos_pack.due_date_offset_days}")
```

Comment:
does this need removing from the config?

Reply:
No, it's used in other places:

```
nigel@Nigels-MacBook-Pro:~/Projects/NFDIV/nfdiv-case-api$ [NFDIV-3795-solicitor_confirm_service] grep -Rl  due_date_offset_days src/main/
src/main//resources/application.yaml
src/main//java/uk/gov/hmcts/divorce/caseworker/service/task/SetDueDateAfterIssue.java
src/main//java/uk/gov/hmcts/divorce/caseworker/service/task/SetReIssueAndDueDate.java
src/main//java/uk/gov/hmcts/divorce/solicitor/event/SolicitorChangeServiceRequest.java
src/main//java/uk/gov/hmcts/divorce/common/service/task/SetSubmissionAndDueDate.java
```